vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -382,7 +382,7 @@ export default {
         {
           id: 9,
           option: "Other",
-          hint: "Please specify referral source",
+          hint: "Other referral source",
```

Comment:
you can change the font-size in mobile view rather than changing the text.

Reply:
I think this hint is better than the previous one. Changed text-size